### DISCUSSION TITLE
Intégrer son compteur électrique ( téléinfo ) dans Home Assistant - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#11] [Comments]
# Intégrer son compteur électrique ( téléinfo ) dans Home Assistant - Haade.fr

téléinformation dans homeassistant comment intégrer son compteur edf

https://haade.fr/fr/blog/integrer-son-compteur-electrique-teleinfo-dans-home-assistant

**Comment by mikebzh44** [comment]
Bonjour.

je pense que je vais acheter 2 de ces petits modules car j'ai un Linky en mode Standard dans mon tableau général et un SAGEM en mode Historique pour ma production solaire (j'ai branché le compteur à l'envers ce qui fait que la consommation mesurée est en fait ma production).

Dans les différents YAML donnés en exemple, je ne trouve pas le tag EAIT (qui correspond à la Energie active injectée totale sur le réseau quand la production est supérieure à la consommation) mais je suppose qu'il n'y a aucune raison pour ne pas pouvoir l'ajouter comme les autres tags.

Une confirmation de votre part serait possible ?

**Reply by haade-administrator** [reply]
Bonjour, oui le tag EAIT est apparemment pris en charge par Enedis pour les **comptes producteurs**, si tu as un **compte consommateur** l'info ne remontera pas, si tu veux essayer remplis une platform EAIT et regarde si les infos remontent tiens moi au jus.
page 19 [https://www.enedis.fr/media/2035/download]( https://www.enedis.fr/media/2035/download)

```
  - platform: teleinfo
    tag_name: "EAIT"
    name: "Energie active injectée totale "
    unit_of_measurement: "Wh"
    icon: mdi:gauge
```

**Reply by mikebzh44** [reply]
Je viens de commander les 2 PiTInfo, je verrai à réception

**Reply by mikebzh44** [reply]
Le compteur d'injection fonctionne très bien avec la balise EAIT

**Reply by haade-administrator** [reply]
Nickel je vais rajouter l'info dans le code

**Comment by grumpy56** [comment]
Bonjour,
Je débute avec HA et EspHome et j'ai réussi a connecter mon Linky et a remonter les infos dans HA avec succès. Merci de ton tuto.
Je souhaiterais maintenant remonter les infos de mon second Linky (injection prod panneaux solaires) en utilisant le même 8266 avec une seconde interface. ...
Et là je ne sais pas comment compléter le code du 8266 .... un coup de main me serait utile.
D'avance, merci d'éclairer la lanterne d'un débutant.

**Comment by Dubixgit** [comment]
Bonjour et merci pour ce tuto très clair !

Je dispose d'un compteur "non-linky" et j'ai souscrit à l'abonnement Tempo.
Cette solution permet-elle de récupérer les différentes valeurs d'énergie consommée les jours bleus, blancs et rouges en HC et HP ?
Si non, est-ce envisageable seulement en adaptant  le code ?

Merci !
Manu

**Comment by Singman33** [comment]
Bonjour,
Tout fonctionnait parfaitement jusqu'à récemment.
Mais aujourd'hui, je n'ai plus l'affichage dans HA de l'intensité, l'intensité max, l'intensité souscrite et le type d'abonnement.
Pourtant dans les logs ESPHome, les valeurs sont bien fournies par le module.
J'en conclus que le module "teleinfo" de HA a subit des modifications ... 
Des idées ?

**Comment by grumpy56** [comment]
Vérifié si il n'y a pas eu de changement de noms des variables suite à une
mise à jour. Ça m'est arrivé il y a qq semaines ...

**Comment by Thierry13011** [comment]
Merci beaucoup pour ton travail

**Reply by haade-administrator** [reply]
Merci
